### PR TITLE
Fixes on workflow

### PR DESCRIPTION
**Pull request by ebocher**
This PR fixes several issues : #364, #370, #381 , #370, #383

This PR identifies some new issues : 
-  on height values when they are derived from level type
See commented tests in BDTopo and OSM

- the algorithm to find the  type and main_use must  be improved in OSM to not to select only the first value found in the types specified by the user. See https://github.com/orbisgis/geoclimate/issues/383

### CODE REVIEW

File: osm/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/osm/WorkflowOSM.groovy [hunk 1]
```diff
@@ -216,7 +223,7 @@ create {
                             if(!osmprocessing.execute(h2gis_datasource:h2gis_datasource,
```
Comment (j3r3m1) [inline osm/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/osm/WorkflowOSM.groovy:223]: do we add areaBuildingTypeAndComposition and FloorAreaTypeAndComposition in the processing_parameters ?

File: bdtopo_v2/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/ProcessingChainBDTopoTest.groovy [hunk 2]
```diff
@@ -225,7 +225,7 @@ class ProcessingChainBDTopoTest extends ChainProcessAbstractTest{
         //configFile =getClass().getResource("config/bdtopo_workflow_folderinput_dboutput.json").toURI()
         //configFile =getClass().getResource("config/bdtopo_workflow_dbinput_dboutput.json").toURI()
         IProcess process = BDTopo.WorkflowBDTopo_V2.workflow
-        assertTrue(process.execute(configurationFile: configFile))
+        assertTrue(process.execute(configurationFile: "/home/ebocher/applications/geoclimate/bdtopo_workflow_dbinput_dboutput_erwan.json"))
```
Comment (j3r3m1) [inline bdtopo_v2/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/ProcessingChainBDTopoTest.groovy:228]: remove the path ?

Comment (ebocher) [reply bdtopo_v2/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/ProcessingChainBDTopoTest.groovy:228]: good catch

**Comment by ebocher** [comment]
Thanks @SPalominos

**Pull request merged**